src/core/mdn-annotation.js

```diff
@@ -31,6 +31,14 @@ const MDN_BROWSERS = {
   webview_android: "WebView Android",
 };
 
+const localizationStrings = {
+  en: {
+    inAllEngines: "This feature is in all current engines.",
+    inSomeEngines: "This feature is in less than two current engines.",
```

Comment:
"This feature has poor support"? The number should be shown next anyway.

Reply:
s/poor/limited maybe.

Reply:
> "This feature has poor support"? The number should be shown next anyway.

"This feature has limited support (n engines)"?

Reply:
The support table should show how its limited, so maybe no need for "n engines" part.